README.md

```diff
@@ -103,3 +103,37 @@ Run Storybook in the project root
 ```
 npm run storybook
 ```
+
+## React Testing Library (RTL) && Jest
+
+Install dependencies
```

Comment:
We need to remove the installation from our dependencies. 1 installation is all a new user would need to run. Then the specific commands would be needed for the docs.

Reply:
@shashilo Thanks for clarifying. I removed the `pnpm i` since only 1 installation is needed and left specific commands on how to run tests.